tools/accuracy_checker/configs/hbonet-0.25.yml

```diff
@@ -0,0 +1,66 @@
+models:
+  - name: hbonet-0.25
+
+    launchers:
+      - framework: onnx_runtime
+        model: public/hbonet-0.25/hbonet-0.25.onnx
+        adapter: classification
+        inputs:
+          - name: data
+            type: INPUT
+            shape: 1,3,224,224
+
+    datasets:
+      - name: imagenet_1000_classes
+        reader: opencv_imread
+
+        preprocessing:
+          - type: bgr_to_rgb
+
+          - type: resize
+            size: 256
+            aspect_ratio_scale: greater
+            interpolation: BILINEAR
+
+          - type: crop
+            size: 224
+
+          - type: normalization
+            std: 255
+
+          - type: normalization
+            mean: (0.485, 0.456, 0.406)
+            std: (0.229, 0.224, 0.225)
+
+  - name: hbonet-0.25
+
+    # list of launchers for MobileNetV2.
```

Comment:
please remove, i think it is copy-pasted accidentally

Reply:
done